lib/crawl.js

```diff
@@ -100,8 +115,8 @@ function crawl(url, flags) {
             urls = limitSimilar(urls, flags.limitSimilar);
         }
         defaultConf.scenarios = urls;
-        const path = dirname(outfile);
-        mkpath(path, mkpathErr => {
+        const outPath = dirname(outfile);
```

Comment:
Needed to use the `path` package so renamed this

Reply:
👍